client/app/app.vue

```diff
@@ -1,13 +1,77 @@
 <template lang="pug">
     div(role="application")
-        ul
-            li: router-link(to="/foo") foo
-            li: router-link(to="/bar") bar
+        aside(role="navigation")
+            h4 {{ 'my_accounts title' | t }}
+            ul
+                li
+                    router-link(to="/discovery")
+                        svg: use(:xlink:href="require('./assets/sprites/icon-discovery.svg')")
+                        | {{ 'my_accounts discovery title' | t }}
+
+                li
+                    router-link(to="/category")
+                        svg: use(:xlink:href="require('./assets/sprites/icon-category.svg')")
+                        | {{ 'my_accounts category title' | t }}
+
+                li
+                    router-link(to="/connected")
+                        svg: use(:xlink:href="require('./assets/sprites/icon-connected.svg')")
+                        | {{ 'my_accounts connected title' | t }}
 
         router-view
 </template>
 
 
 <style lang="stylus">
-@import "../node_modules/normalize.css/normalize.css";
+    @import '../node_modules/normalize.css/normalize.css'
+    @import './styles/base/_normalize'
+    @import './styles/base/_colors'
+    @import 'cozy-ui'
+
+    [role=application]
+        @extend $app-2panes-toolbar
+        background-color: $grey-01-alpha
+
+        main
+            padding: 2.5em 3em
+
+        h1
+            font-size: 2em
+            margin: 0 0 1em
+
+        h4
+            font-weight: normal
+            font-size: 1.5em
+            padding: 1em 1.5em
+            margin: 0
+
+
+      [role="navigation"]
```

Comment:
Maybe we can simply rely on the `aside` tag (or eventually be more specific with a `[role=application] > aside` and reduce the nesting of one level, it'll reduce the specificity :)